PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -620,7 +620,17 @@ def miniAOD_customizeData(process):
     from PhysicsTools.PatAlgos.tools.coreTools import runOnData
     runOnData( process, outputModules = [] )
     process.load("RecoCTPPS.Configuration.recoCTPPS_cff")
+    process.load('L1Trigger.L1TGlobal.simGtExtFakeProd_cfi')
+    process.simGtExtFakeProd.bxFirst = cms.int32(-2)
+    process.simGtExtFakeProd.bxLast = cms.int32(2)
+    process.simGtExtFakeProd.setBptxAND = cms.bool(True)
+    process.simGtExtFakeProd.setBptxPlus = cms.bool(True)
+    process.simGtExtFakeProd.setBptxMinus = cms.bool(True)
+    process.simGtExtFakeProd.setBptxOR = cms.bool(True)
+    process.simGtExtFakeProd.tcdsRecordLabel = cms.InputTag("tcdsDigis","tcdsRecord")
     task = getPatAlgosToolsTask(process)
+    from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
+    run2_miniAOD_UL.toModify(task, func=lambda t: t.add(process.simGtExtFakeProd))
```

Comment:
Should this line be just moved to after line 635, so that we don't need to call "from ... run2_miniAOD_UL" twice?

Reply:
yes